core/lib/common/worker.js

```diff
@@ -394,41 +400,46 @@ module.exports = class Worker {
 		// this will be torn down at the end of the tests when the core is destroyed
 		let argsClient = [
 			`tcp-listen:${dutPort},reuseaddr,fork`,
-			`system:ssh ${this.workerUser}@${this.workerHost} -p ${this.workerPort} -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -i ${this.sshKey} ${this.sshPrefix}/usr/bin/nc localhost ${workerPort}`,
+			`system:ssh ${this.workerUser}@${this.workerHost} -p ${this.workerPort} -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null ${cloudPrefix}/usr/bin/nc localhost ${workerPort}`,
 		];
 
 		let tunnelProcClient = spawn(`socat`, argsClient);
 	}
 
 	// create tunnels to relevant DUT ports to we can access them remotely
 	async createSSHTunnels(target) {
-		if (!this.url.includes(`localhost`)) {
-			const DUT_PORTS = [
-				48484, // supervisor
-				22222, // ssh
-				2375, // engine
-			];
-			let workerPort = 8888;
-			for (let port of DUT_PORTS) {
-				console.log(`creating tunnel to dut port ${port}...`);
-				await this.createTunneltoDUT(target, port, workerPort);
-				workerPort = workerPort + 1;
-			}
+		const DUT_PORTS = [
+			48484, // supervisor
+			22222, // ssh
+			2375, // engine
+		];
+		let workerPort = 8888;
+		for (let port of DUT_PORTS) {
+			console.log(`creating tunnel to dut port ${port}...`);
+			await this.createTunneltoDUT(target, port, workerPort);
+			workerPort = workerPort + 1;
```

Comment:
Remind me why the core can't communicate with the DUT directly anymore? If the worker is in bridge network but still able to create SSH tunnels to the DUT, that stands to reason that the core service, on the same bridge network, can also reach the DUT?

I know we talked about this but I'm still not clear why the SSH tunnels are required for QEMU workers.